tests/testthat/test-yeo-johnson.R

```diff
@@ -0,0 +1,105 @@
+suppressPackageStartupMessages(source(here::here("R", "load_all.R")))
+
+test_that("Yeo-Johnson transformation inverts correctly", {
+  expect_true(
+    map_lgl(seq(-5, 5, 0.1), function(lambda) {
+      map_lgl(seq(0, 10, 0.1), \(x) abs(yj_inverse(yj_transform(x, lambda), lambda) - x) < 0.00001) %>% all()
```

Comment:
```suggestion
      map_lgl(seq(-10, 10, 0.1), \(x) abs(yj_inverse(yj_transform(x, lambda), lambda) - x) < 0.00001) %>% all()
```
Should be robust to negative values of `x` (it does pass this thankfully). Also want to note that the special lambda values of -1, 0, and 1 are all covered exactly already.